joomla/joomla-cms
# [4.0] Rewrite indexer in vanilla Javascript

### Summary of Changes

Remove all jQuery from `indexer.js` (apart from 1 line which I'm not sure about)

### Testing Instructions

Ensure the smart search indexer still works.

There is an open issue (https://github.com/joomla/joomla-cms/issues/18861) by which the index doesn't work with sample data on a multilang site. So testing can be done on a installation with no sample data.

@franz-wohlkoenig Can you check if this solves the issue you raised? Don't think it will but just incase.
